app/src/main/java/org/kinecosystem/kinit/view/earn/QuestionDualImageFragment.kt

```diff
@@ -14,6 +14,10 @@ import org.kinecosystem.kinit.viewmodel.earn.QuestionDualImageViewModel
 
 class QuestionDualImageFragment : BaseFragment() {
 
+    interface OnUpdateObservable {
+        fun onValueChanged()
```

Comment:
I would try to be more descriptive in the naming here, it's not clear what value has been changed etc.